public/css/campaigns.css

```diff
@@ -42,6 +42,12 @@ h3, h4, h5, p {
     max-width: 100vw;
 }
 
+.campaign-date {
```

Comment:
Looks OK for now, but let's review with design to finalize. (we can merge for now and have Hendrika look at it)